### PR TITLE
Feature/1380 publishing page adding credentials to publish test live fe improvements

### PR DESCRIPTION
**Pull request by stevenburnell-moj**
**Some of this:**
Screenshot 2021-03-30 at 18 09 33

**and this:**
Screenshot 2021-03-30 at 18 09 42

**and a little bit of this:**
Screenshot 2021-03-30 at 18 09 48

### CODE REVIEW

File: app/views/publish/index.html.erb [hunk 1]
```diff
@@ -18,14 +18,14 @@
...
-    <span>
-      <%= PublishServicePresenter.hostname_for(deployment_environment: 'dev', view: self) %>
-    </span>
-
+    <%# TODO: Need an indicator from BE code that shows whether this is first-time publish   %>
+    <%#       Currently have hardcoded 'false' value to avoid dialog showing but this should %>
+    <%#       be dynamic to control dialog visibility.                                        %>
     <%= form_for(@publish_service_creation_dev,
         url: publish_index_path(service.service_id),
-        html: { id: 'publish-form-dev', class: 'publish-form' }) do |f| %>
```
Comment (stevenburnell-moj) [inline app/views/publish/index.html.erb]: @tomas-stefano 
Notice how I've hardcoded a value here.
`data: { firstpublish: false }`

If we (and by that I mean you - hehe! ) can make this dynamic then we will have a completed controlling mechanism for showing the '15 minute' dialog message on initial publish (make it true, publish a form, and you will see it).

Let me know if that's clear as mud.

File: app/views/publish/index.html.erb [hunk 2]
```diff
@@ -42,13 +42,12 @@
...
 
-    <span>
-      <%= PublishServicePresenter.hostname_for(deployment_environment: 'production', view: self) %>
-    </span>
+    <p><%= PublishServicePresenter.hostname_for(deployment_environment: 'production', view: self) %></p>
 
+    <%# TODO: Hardcoded 'firstpublish' (see comment above) %>
     <%= form_for(@publish_service_creation_production,
         url: publish_index_path(service.service_id),
-        html: { id: 'publish-form-live', class: 'publish-form' }) do |f| %>
```
Comment (stevenburnell-moj) [inline app/views/publish/index.html.erb]: @tomas-stefano 
Second hardcoded value that controls things for Production environment. Same thing as previous comment applies.

File: app/views/publish/index.html.erb (hunk 1 above)
Comment (brenetic) [reply app/views/publish/index.html.erb]: surely you mean `first_publish`? 😜

Comment (tomas-stefano) [reply app/views/publish/index.html.erb]: Cool, agree with above on `first_publish`. Happy to merge this PR and add the dynamic in a separate one. :)

**Review by tomas-stefano — APPROVED** [review]

**Pull request merged**